src/hooks/user.js

```diff
@@ -459,7 +459,29 @@ export const UserProvider = ({ children }) => {
             return AsyncStorage.getItem(key)
         }
         return JSON.parse(await AsyncStorage.getItem(key))
-    }
+		}
+	
+		const pingServer = async (ip, callback) => {
```

Comment:
Não consegui compreender a utilidade dessa função, não utilizar `this` também porque esse componente é funcional.